devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -136,6 +141,13 @@ def updateFinalState() {
 	}
 }
 
+def batteryPercentageEventHandler(batteryLevel) {
+	if (batteryLevel != null) {
+		batteryLevel = Math.min(100, Math.max(0, batteryLevel))
+		sendEvent([name: "battery", value: batteryLevel, unit: "%", descriptionText: "{{ device.displayName }} battery was {{ value }}%", isStateChange: true])
```

Comment:
I don't think you need to set `isStateChange: true`

Reply:
removed